packages/elastic-apm-synthtrace/src/lib/entity.ts

```diff
@@ -26,4 +26,13 @@ export class Entity<TFields extends Fields> {
 
     return this;
   }
+
+  setFields(fields: TFields) {
```

Comment:
I'm just now noticing we already have `defaults` - that should suffice here no? if not, can you rename `setFields` to just `set`?

Reply:
yeah, we can you defaults